sass/bhclub.scss

```diff
@@ -94,31 +109,118 @@ nav {
         width: 75px;
         max-width: 100%;
         margin: 0px 0px 20px;
-    }
-
-    a.btn-light {
-        display: inline-block;
+        border: none;
     }
 
     .nav-link {
-        font-size: 1.5rem;
-        margin: 10px 0;
+        font-size: 1.25rem;
+        margin: 10px 0 14px;
+        width: fit-content;
+
+        &:hover {
+            margin-bottom: 10px;
+        }
     }
 
-    .social-link {
-        font-size: 1.125rem;
-        margin: 10px 0;
+    #navbar-middle, #navbar-top {
+        margin-bottom: auto;
     }
 }
 
+.mobile-grayout {
+    position: fixed;
+    width: 100%;
+    height: 100%;
+    top: 0;
+    left: 0;
+    background-color: black;
+    opacity: 25%;
+    z-index: 2;
+    cursor: pointer;
+    display: none;
+}
+
 // Main Content
 main {
-    padding: 3.5rem;
+    padding: 1.75rem 3.5rem;
     padding-left: calc(25% + 3.5rem); // pushes it outside of the nav while allowing for backgrounds to stretch underneath
 }
 
 section {
-    padding: 5rem 0;
+    padding: 1.75rem 0;
+    min-height: 100vh;
+    display: flex;
+    flex-flow: column nowrap;
+    justify-content: center;
+}
+
+// About Section
+#about > p {
+    width: 60%;
+}
+
+.card {
+    border: 2px solid $blue;
+    border-radius: 6px;
+    margin-top: 1rem;
+    width: 75%;
+
+    .card-header {
+        font-weight: 600;
+        color: $blue;
+        text-align: left;
+        font-size: 1.25rem;
+        background-color: white;
+        border: none;
+        width: 100%;
+        padding: 0.625rem 1.25rem;
+        cursor: pointer;
+        transition: all 1s;
```

Comment:
I think this is also transitioning the `border-radius` which imo would create a better effect if it was faster

Reply:
I think I found a way to leave border-radius out of the transition (or I am reaching the point where I'm imagining that there are things changing)